subgraphs/erc721-holders/src/mappings/collection.ts

```diff
@@ -253,3 +262,21 @@ function createTransfer(event: TransferEvent): Transfer {
 
   return transfer;
 }
+
+function updateCollectionDetails(
```

Comment:
utility function for updating the name/symbol of a collection created via registry (collection doesn't exist until the contract is deployed and used so name/symbol will be null)